xmeffects.js

```diff
@@ -287,3 +293,4 @@ var effects_t1 = [  // effect functions on tick 1+
   eff_unimplemented,  // y
   eff_unimplemented   // z
 ];
+})(window || {});
```

Comment:
Yea that's basically what I was thinking. One could swap it out for some other module implementation as a fallback if `window` is undefined (running in a non-browser)